Backend/Remora.Discord.Core/Optional.cs

```diff
@@ -60,20 +60,18 @@ namespace Remora.Discord.Core
             }
         }
 
-        // Technically, this is a lie, but since the nullability of the type is reliant on the actual generic type and
-        // its annotations, this warning can be disabled.
-        #pragma warning disable CS8775
```

Comment:
No warning needs to be disabled here. Works fine on NetFx and NetCore with it on. Comment is also removed since this the associated value has nothing to do with the nullability of the type, only whether it is logically present.